### PR TITLE
project/jupyter: update nteract dependencies

### PR DESCRIPTION
**Pull request by haraldschilly**
in particular to get the fix for https://github.com/nteract/kernelspecs/issues/28

... I haven't tested it, this is more or less just a reminder that we need to update this.

### CODE REVIEW

**Comment by haraldschilly** [comment]
not good, running the tests with the updated libs gives 4 errors:

```
1) compute 2/3 using the python3 kernel --  evaluate 2/3:
     Error: Timeout of 5000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
 
  2) get input using the python3 kernel --  do it:
     Error: Timeout of 10000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
 
  3) test the non-sage R kernel -- evaluate sd(c(1,2,5)):
     Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
  
 
  4) test the sage R kernel -- evaluate sd(c(1,2,5)):
     Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

File: src/smc-project/jupyter/package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "dependencies": {
     "enchannel-zmq-backend": "^2.6.1",
```
Comment (rgbkrk) [inline src/smc-project/jupyter/package.json]: You'll want to bring this up to `"enchannel-zmq-backend": "^4.0.0",`

Comment (rgbkrk) [inline src/smc-project/jupyter/package.json]: (I just published this too).

We're about to have this setup where these dependent projects are together in one monorepo, though still separate node modules.

Comment (haraldschilly) [inline src/smc-project/jupyter/package.json]: ok, thank you, and a consistent monorepo sounds good (to eliminate the combinatorial explosion)

**Comment by haraldschilly** [comment]
I updated `enchannel-zmq-backend` as well, increased some timeouts, and now only one test fails: `test the sage R kernel -- evaluate sd(c(1,2,5))`

not sure what's going on, but since all the other (similar) tests pass, it look ok.

also, going back to the old version, 12 tests fail...

**Comment by williamstein** [comment]
That R test used to fail a lot...

**Pull request merged**